src/main/java/io/cdap/plugin/http/source/common/BaseHttpSourceConfig.java

```diff
@@ -961,8 +965,10 @@ PAGINATION_INDEX_PLACEHOLDER, getPaginationType()),
   }
 
   private boolean refreshTokenGrantType() {
```

Comment:
why do we need this method, I can directly check for the condition where ever we are using this. In case we will add multiple grant types in future, it will create issues

Reply:
checking the condition directly now.